src/org/labkey/test/util/FileBrowserHelper.java

```diff
@@ -145,6 +148,41 @@ public void selectFileBrowserItem(@LoggedParam String path)
         }
     }
 
+    public void sortFileBrowserColumn(String columnText, SortDirection sortDirection)
+    {
+        Locator headerLoc = Locator.tagWithClass("div", "fbrowser")
+                .descendant(Locator.tagWithClass("div", "x4-column-header")
+                .withChild(Locator.tagWithClass("div", "x4-column-header-inner")
+                .withChild(Locator.tagWithClass("span", "x4-column-header-text").withText(columnText))));
+
+        WebElement headerElement = headerLoc.findElement(getDriver());
+
+        // often initial state is unsorted, first click sets it to ASC
+        if (!getSortDirection(headerLoc).equals(sortDirection))
+        {
+            doAndWaitForFileListRefresh(()-> headerElement.click());
+        }
+        // if we intended DESC, it may require another click
+        if (!getSortDirection(headerLoc).equals(sortDirection))
+        {
+            doAndWaitForFileListRefresh(()-> headerElement.click());
+        }
```

Comment:
Flip `equals` checks to prevent NPE from unsorted columns (see my comment on `getSortDirection`).

Reply:
I've figured out a way to fly out the header sort menu; clicking that seems better